lib/LLVMPasses/LLVMSwiftAA.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "swift/LLVMPasses/Passes.h"
```

Comment:
We should be getting `swift/LLVMPasses/Passes.h` transitively from this include. Is this note the case?

Reply:
### swift-swift-5.2.4-RELEASE/include/swift/LLVMPasses/Passes.h:
#ifndef SWIFT_LLVMPASSES_PASSES_H                                                                                                                                         
#define SWIFT_LLVMPASSES_PASSES_H                                                                                                                                         
                                                                                                                                                                          
#include "swift/LLVMPasses/PassesFwd.h"                                                                                                                                   
#include "llvm/Analysis/AliasAnalysis.h"                                                                                                                                  
#include "llvm/Pass.h"                                                                                                                                                    
                                                                                                                                                                          
namespace swift {   


### llvm-project-llvmorg-10.0.1/llvm/include/llvm/Pass.h:
#ifndef LLVM_PASS_H                                                                                                                                                       
#define LLVM_PASS_H                                                                                                                                                       
                                                                                                                                                                          
#include "llvm/ADT/StringRef.h"                                                                                                                                           
#include <string>                                                                                                                                                         
                                                                                                                                                                          
namespace llvm {  

### llvm-project-llvmorg-10.0.1/llvm/include/llvm/Analysis/AliasAnalysis.h:
#ifndef LLVM_ANALYSIS_ALIASANALYSIS_H                                                                                                                                     
#define LLVM_ANALYSIS_ALIASANALYSIS_H                                                                                                                                     
                                                                                                                                                                          
#include "llvm/ADT/DenseMap.h"                                                                                                                                            
#include "llvm/ADT/None.h"                                                                                                                                                
#include "llvm/ADT/Optional.h"                                                                                                                                            
#include "llvm/ADT/SmallVector.h"                                                                                                                                         
#include "llvm/Analysis/MemoryLocation.h"                                                                                                                                 
#include "llvm/Analysis/TargetLibraryInfo.h"                                                                                                                              
#include "llvm/IR/Function.h"                                                                                                                                             
#include "llvm/IR/Instruction.h"
#include "llvm/IR/Instructions.h"
#include "llvm/IR/PassManager.h"
#include "llvm/Pass.h"
#include <cstdint>
#include <functional>
#include <memory>
#include <vector>

namespace llvm {

Reply:
https://github.com/apple/swift/blob/master/include/swift/LLVMPasses/Passes.h#L19